assets/stylesheets/desktop.css

```diff
@@ -0,0 +1,505 @@
+/******************* Header *********************/
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  overflow-x: hidden;
+}
+
+.mobile-header {
+  display: none;
+}
+
+header {
+  position: fixed;
+  width: 100%;
+  top: 0;
+  margin-top: 0;
+  font-family: 'Lato', sans-serif;
+  font-size: 14px;
+}
+
+header a {
+  text-decoration: none;
+}
+
+header nav a {
+  color: #272a31;
+}
```

Comment:
- [ ] [OPTIONAL]: You could consider adding this color (and all the other ones) to a variable, considering you are going to keep reusing it. You can read more about it [here](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties).

![image](https://user-images.githubusercontent.com/66967274/147238111-22fcb548-a988-4a67-81be-a8dba343ed7e.png)